README.md

```diff
@@ -36,6 +36,9 @@ $ go get -d github.com/kinvolk/kube-spawn
 $ cd $GOPATH/src/github.com/kinvolk/kube-spawn
 $ make all
 
+# (optional) Install binaries under a system directory
```

Comment:
Can you say where they will be installed and how it can be changed?

Reply:
@schu Done. Added some comments.